app/src/main/java/org/shadowice/flocke/andotp/Utilities/EntryThumbnail.java

```diff
@@ -150,6 +150,7 @@
         Wallabag(R.drawable.thumb_wallabag),
         Wallet(R.drawable.thumb_wallet),
         Wordpress(R.drawable.thumb_wordpress);
```

Comment:
This line needs to end with `,` instead of `;`.

Reply:
Right, thanks!